README.md

```diff
@@ -118,11 +118,12 @@ nile deploy contract --alias my_contract
 
 A few things to notice here:
 
-1. `nile deploy <contract_name>` looks for an artifact with the same name
-2. This created a `localhost.deployments.txt` file storing all data related to my deployment
-3. The `--alias` parameter lets me create a unique identifier for future interactions, if no alias is set then the contract's address can be used as identifier
-4. By default Nile works on local, but you can use the `--network` parameter to interact with `mainnet`, `goerli`, and the default `localhost`.
-5. By default, the ABI corresponding to the contract will be registered with the deployment. To register a different ABI file, use the `--abi` parameter.
+- `nile deploy <contract_name>` looks for an artifact with the same name.
+- This created a `localhost.deployments.txt` file storing all data related to my deployment.
```

Comment:
We should maintain consistency in grammar for the README: 

```suggestion
- This creates or updates the` localhost.deployments.txt` file storing all data related to deployments.
```

There are more places where this happens, like the next sentence, where instead of "lets me create" I would say something like "allows creating".

I think we should maintain third-person singular, but if not, at least we should be consistent.

I would prioritize an issue for refactoring deeply the README for this, or the next milestone @martriay @andrew-fleming .

Reply:
Mmm yeah, I'm not a fan of the first-person. I'd say third-person singular or even second person ("lets you create"). I agree on prioritizing a README refactor.

Reply:
I saw now I wrote third-person singular where I meant third-person present, sorry.

Reply:
Haha I rewrote the same thing. I understood what you meant :)